src/span.rs

```diff
@@ -1,74 +1,55 @@
-/// A macro for creating a raw (unstyled) [`Span`] using formatting syntax.
+/// A macro for creating a [`Span`] using formatting syntax.
 ///
-/// `raw!` is similar to the [`format!`] macro, but it returns a [`Span`] instead of a `String`.
+/// `span!` is similar to the [`format!`] macro, but it returns a [`Span`] instead of a `String`.
+///
+/// The first argument is a [`Style`] or any type that is convertible to [`Style`] (e.g. [`Color`]).
```

Comment:
"If there is a semicolon, then ... otherwise, the Span will be created as a raw span (with style set to Style::default()).

Reply:
Resolved in https://github.com/ratatui-org/ratatui-macros/pull/30/commits/f6cea575bcac9f289fb3d1c6a44e057cbeea795e